### DISCUSSION TITLE
Clipboard contents being copied into users local machines

### DISCUSSION THREAD

**Discussion by blunt4286** [rustdesk/rustdesk#5385] [General]
### Bug Description

I copy something on my local machine, it gets copied into the local clipboard of other connected users.
For example, we have 3 users connected to the same machine using rustdesk.
If user A is working on remote machine via rustdesk, and user B copies something on his local machine, if user A pastes on his local machine, he will get the copied content from user Bs local clipboard

### How to Reproduce

Connect 2 users to the same machine via rustdesk.
perform a copy on the first users local machine.
paste on second users local machine, and you will paste the content from first user.

### Expected Behavior

rustdesk should use the users local clipboard.
user A and B should have their own clipboard history, and be able to paste their own things.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.1.9

### Screenshots

![rustdesk](https://github.com/rustdesk/rustdesk/assets/29315611/c945c69e-01dc-4a1d-a357-8d6890d4569f)

In the screenshot attached, the 4 lines of new day were not copied my me, they were copied by a colleague.
I connected via rustdesk and those contents got copied into my local clipboard.
I did not have right-click > Copy or Ctrl+C.

**Comment by blunt4286** [comment]
small update... this happens on client version 1.2.2 as well

**Reply by DenzyS** [reply]
The author, antiviruses swear at rustdesk-1.2.2-x86_64.exe, browsers with built-in antiviruses do not even let you download it. I'm not implying anything, I'm just stating the fact.

**Reply by dinger1986** [reply]
> The author, antiviruses swear at rustdesk-1.2.2-x86_64.exe, browsers with built-in antiviruses do not even let you download it. I'm not implying anything, I'm just stating the fact.

There's a separate thread for this!
